eng/docker-compose/start-published-dms.ps1

```diff
@@ -73,16 +71,16 @@ else {
         docker network create dms
     }
 
-    Write-Output "Starting published DMS"
-    docker compose $files --env-file $EnvironmentFile -p dms-published up -d
-
+    Write-Output "Starting Keycloak first..."
+    docker compose -f keycloak.yml -env-file $EnvironmentFile -p dms-published up -d
     if ($LASTEXITCODE -ne 0) {
-        throw "Unable to start Published Docker environment, with exit code $LASTEXITCODE."
+        throw "Failed to start Keycloak. Exit code $LASTEXITCODE"
     }
 
+    Write-Output "Waiting for Keycloak to initialize..."
     Start-Sleep 20
```

Comment:
@semalaiappan 
The sleep time increased to 45 seconds now. Please reduce the wait time for keycloak setup and try. Same for the start-local-dms as well.

Reply:
I am getting this error if i reduce sleep time